presto-hive/src/main/java/com/facebook/presto/hive/HivePageSourceProvider.java

```diff
@@ -652,14 +656,14 @@ public int getIndex()
         /**
          * @param columns columns that need to be returned to engine
          * @param requiredInterimColumns columns that are needed for processing, but shouldn't be returned to engine (may overlaps with columns)
-         * @param partitionSchemaDifference map from hive column index to hive type
+         * @param tableToPartitionMapping   table to partition mapping
```

Comment:
reduce blank between tableToPartitionMapping and table

Reply:
Done 👍🏼